docs/sample_config.yaml

```diff
@@ -142,6 +142,17 @@ presence:
 #
 #limit_profile_requests_to_users_who_share_rooms: true
 
+# User Directory Updates
+#
+# If specified, a worker with that name will be the only one able to update
+# the user directory. Important for querying shared rooms. If the worker is
+# specified, but not running, the user directory may become outdated.
+#
+# Defaults to null, meaning the main process will handle this.
+#
+# Set to false to make sure no process will handle the user_directory.
```

Comment:
so it can be a string, or a boolean, or `null`? ew. can we find a way to limit it to just one type or `null`?

Reply:
We could, the extra type was suggested by @turt2live to basically "disable" the config, though maybe we can change that to be `""`, an empty string.